tests/kafkatest/services/kafka/kafka.py

```diff
@@ -548,7 +548,7 @@ def security_config(self):
                 interbroker_security_protocol_to_use=SecurityConfig.PLAINTEXT
 
             self._security_config = SecurityConfig(self.context, security_protocol_to_use, interbroker_security_protocol_to_use,
```

Comment:
sorry if this is a silly question, but does this file need to change in this PR? seems like we could do this in the follow-on?